packages/ffe-form-react/src/Input.spec.js

```diff
@@ -22,6 +25,15 @@ describe('<Input />', () => {
         expect(wrapper.hasClass('ffe-input-field')).toBe(true);
     });
 
+    it('passes refs', () => {
```

Comment:
Maybe expand the description here with why this is needed? Future Developer should be happy with not having to search through the Git log 😉

Reply:
I mean, it's a feature / quirk of React - not sure if I want to go around and explain all of those. I'll add a comment, sure, but I don't think it's needed in a general sense.